include/signUp.php

```diff
@@ -2,14 +2,25 @@
 
 function insertUser($username, $password, $name, $email){
    dbQuery(
-       'INSERT INTO users (username, password, email, name)
+       '
+       INSERT INTO users (username, password, email, name)
         VALUES(:username, :password, :email, :name)',
     [
-        'username'=>$_REQUEST['username'],
-        'password'=>$_REQUEST['password'],
-        'name'=>$_REQUEST['name'],
-        'email'=>$_REQUEST['email'],
+        'username'=>$username,
+        'password'=>$password,
+        'name'=>$name,
+        'email'=>$email,
     ]
 
 );
 }
+
+function displayErrors(){
```

Comment:
This function is fine, but I just want to point out that it doesn't seem specific to the signup process. Really any form could use this.